src/app/_components/StatisticCard.tsx

```diff
@@ -0,0 +1,30 @@
+import { Icon, type IconProps } from '@/components/Icon'
+
+export type StatisticCardProps = {
+  icon: IconProps['component']
+  value: number
+  description: string
+}
+
+export function StatisticCard({
+  icon,
+  value,
+  description,
+}: StatisticCardProps) {
+  return (
+    <li className="flex min-h-32 rounded-lg border border-brand-300 p-1">
+      <figure className="grid w-1/3 place-items-center rounded bg-brand-500">
+        <Icon component={icon} color="brand-200" size={40} />
+      </figure>
+
+      <div className="flex w-2/3 place-items-center p-4">
+        <div className="space-y-2">
+          <span className="text-4xl font-light text-brand-100">
+            {value.toLocaleString()}+
+          </span>
+          <p className="text-sm leading-4 text-brand-200">{description}</p>
```

Comment:
in the designs, font size is 12px?

Reply:
It's very small. I'll talk to Filipa! Thank you.